dev-guide/host-port-registry.md

```diff
@@ -64,6 +64,7 @@ Ports are assumed to be used on all nodes in all clusters unless otherwise speci
 
 | Port      | Process                                         | Protocol | Control-plane only | Owning Team   | Since | Notes                                                                                                            |
 |-----------|-------------------------------------------------|----------|--------------------|---------------|-------|------------------------------------------------------------------------------------------------------------------|
+| 53        | coredns                                         |          |                    | opnet         | 4.7   | On-prem IPI only                                                                                                 |
```

Comment:
Technically "on-prem" is too narrow because there are cloud deployments where it also exists